static/css/styles.css

```diff
@@ -0,0 +1,3 @@
+div{
+    color: deepskyblue;
+}
```

Comment:
It's better to always add a newline at the end of the file. You can read why this is the case [here](https://stackoverflow.com/a/729795).